shop/src/main/java/ro/msg/learning/shop/repository/IRepositoryStock.java

```diff
@@ -3,5 +3,10 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import ro.msg.learning.shop.entity.Stock;
 
+import java.util.List;
+
 public interface IRepositoryStock extends JpaRepository<Stock, Long> {
```

Comment:
You should use here StockId instead of Long.

Reply:
Fixed.